docs/faq.md

```diff
@@ -67,15 +70,80 @@ Embed a special version of Semgrep Playground with an `iframe`. The source is `h
 <iframe title="Semgrep example no prints" src="https://semgrep.dev/embed/editor?snippet=ievans:print-to-logger" width="100%" height="432" frameborder="0"></iframe>
 ```
 
+## Usage limits
+
+This section describes usage limits.
+
+:::caution important dates
+* The Semgrep Community tier will be replaced with Team tier on May 23, 2023.
+* All users with any number of contributors are able to access Semgrep Supply Chain, Semgrep Pro Rules, and other paid offerings for free as of this update (May 15, 2023).
+* A grace period for all users with any number of contributors is ongoing from now until July 31, 2023.
+:::
+
+### What is changing about Semgrep tiers?
+
+We are sunsetting the Community tier and offering our Team tier for free to all users, subject to a usage limit of 10 contributors scanning on private repositories. A contributor is someone who has contributed code to a private repository being scanned by Semgrep.
+
+You can now access the Team tier features of Semgrep Supply Chain, Semgrep Code, and Semgrep Cloud Platform features such as SSO. Our highest level of support will still be available through the Enterprise tier. This updated Enterprise tier includes such features as single-tenancy and a secure network tunnel, ideal for organizations with increased infrastructure, reporting, and other custom requirements.
```

Comment:
> This updated Enterprise tier includes such features as single-tenancy and a secure network tunnel, ideal for organizations with increased infrastructure, reporting, and other custom requirements.

For now I think we can remove this and just focus on the Community + Team tier, wdyt?

Reply:
```suggestion
```

Reply:
Removed!